third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -229,6 +231,43 @@ def BufferLoadOp : TT_AMDGPU_Op<"buffer_load", [
     }];
 }
 
+//===----------------------------------------------------------------------===//
+// BufferLoadToLocalOp
+//===----------------------------------------------------------------------===//
+
+def BufferLoadToLocalOp : TT_AMDGPU_Op<"buffer_load_to_local", [
+  AttrSizedOperandSegments,
+  MemoryEffects<[MemRead<GlobalMemory>]>,
+  MemoryEffects<[MemWrite<LocalMemory>]>,
+  TypesMatchWith<"result element type matches the pointed type of ptr", "result", "ptr", "getPointerTypeToElement($_self)">,
+  TypesMatchWith<"infer mask shape from offsets",
+                 "offsets", "mask", "getI1SameShape($_self)",
+                 "(cast<BufferLoadToLocalOp>($_op).getMask() == nullptr) || std::equal_to<>()">,
+]>{
+    let summary = "Load from a scalar base pointer and a tensor offset to local memory";
```

Comment:
s/local/shared/

Reply:
Just in the summary and description right? Because all other ops in `tt` and `ttg` use `local` in their names.

Reply:
Yeah just summary and description is fine.